README.md

```diff
@@ -0,0 +1,71 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My First Capstone-Project (Mobile first)
+
+My portfolio project is prepared for different screen sizes. However, I built it by using "mobile first" approach. 
+
+I used this [template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015).
+
+
+## Built With
+
+- Library : Font Awesome
+- Framework : Boostrap
+- Technologies : Html & CSS & JavaScript
+
+
+## Live Demo
+
+[My First Capstone-Project](https://capstone-one.netlify.app/)
+
+
+### Prerequisites
+
+To run this app on your local computer you need the browser (Google Chrome is the best)
+
+### Setup
+
+- Clone the repository on your local machine
+- Cd into the folder
+
+### Install
+
+- Install VSCode or any code editor you like
+- Install npm by running npm install
+
+### Screenshort for the Mobile version
+
+<img width="634" alt="home-about" src="https://user-images.githubusercontent.com/108806646/191325369-e5e45694-c277-410b-b1de-763b88ad3bd5.png">
+
+
+### Screenshort for the Desktop Version
+
+<img width="960" alt="Untitled" src="https://user-images.githubusercontent.com/108806646/191325160-5e0e98ad-7776-4d55-8387-64191caf5a97.png">
+
+
+## Author
+
+👤 **Oliver Kango**
+
+- GitHub: [@github-Olivier](https://github.com/Olivier-Kango)
+- Twitter: [@twitter-Olivier](https://twitter.com/olivierkango1)
+- LinkedIn: [@LinkedIn-Olivier](https://www.linkedin.com/in/olivier-kango-b990601b8/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
```

Comment:
- [ ] Kindly notice that the README contains an "Acknowledgements" section or similar that gives appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).